lib/do_auth/user.ex

```diff
@@ -100,6 +100,8 @@ defmodule DoAuth.User do
     end)
 
     Mail.confirmation(secret, email, nickname, homebase, opts) |> Mailer.deliver_now!()
+
+    %Result.Ok{ok: pid}
```

Comment:
Added to fulfil the contract

Reply:
Awesome! I'll admit: we didn't run `mix dialyzer` with @ArsenijSerjogin :D